addon/index.js

```diff
@@ -439,12 +439,12 @@ exports.main = function(options) {
     store.clientUUID = require('sdk/util/uuid').uuid().toString().slice(1, -1);
   }
 
+  Metrics.init();
```

Comment:
Moved this up to make sure we logged `browserLoadedTimestamp` before running `onEnable`. Possibly want to revert this, pending the outcome of [this comment](https://github.com/mozilla/testpilot/pull/1186/files#r73460377).